src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -959,6 +985,10 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
   glb_policy->lb_call_timeout_ms =
       grpc_channel_arg_get_integer(arg, (grpc_integer_options){0, 0, INT_MAX});
 
+  arg = grpc_channel_args_find(args->args, GRPC_ARG_GRPCLB_FALLBACK_TIMEOUT_MS);
+  glb_policy->lb_fallback_timeout_ms = 
+      grpc_channel_arg_get_integer(arg, (grpc_integer_options){0, 0, INT_MAX});
```

Comment:
Even if the fallback is enabled by default, I think the behavior will only be different if the resolver actually returns backend addresses.  I don't think it's doing that in any of the existing tests, so I think they should not be affected (but you can certainly check).